Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -238,9 +238,17 @@ extension Docc {
             return outputURL
         }
         
-        /// Defaults to false
-        @Flag(help: "Produce a Swift-DocC Archive that supports a static hosting environment.")
-        public var transformForStaticHosting = false
+        /// A Boolean value that is true if the DocC archive produced by this conversion
+        /// will support static hosting environments.
+        ///
+        /// This value defaults to true but can be explicitly disabled with the
+        /// `--no-transform-for-static-hosting` flag.
+        @Flag(
+            inversion: .prefixedNo,
```

Comment:
Fancy 😄 Does this mean that `--transform-for-static-hosting` is still a valid argument? It would be nice to continue supporting that flag (as a no-op) during a compatibility period.